linera-views/src/batch.rs

```diff
@@ -110,6 +110,12 @@ impl UnorderedBatch {
     }
 }
 
+/// Checks if `key` is matched by any prefix in `key_prefix_set`.
+/// The set `key_prefix_set` must be minimal for the function to works correctly.
+/// That is, there should not be ny two prefixes p1 and p2 such that p1 < p2 for
+/// the lexicographic ordering on `Vec<u8>` entries.
+/// The theorem is that the last element of `key_prefix_set` has to be the one
```

Comment:
I don't understand this sentence.
1) "The theorem is that" -> do you mean to say "It is guaranteed that" ?
2) The second part of the sentence seems to contradict the beginning of the summary.

Reply:
Ok, I corrected the statement.